src/examples/recognised-overseas-pension-schemes-notification-list/example/index.njk

```diff
@@ -0,0 +1,87 @@
+---
+  layout: layout-example.njk
+---
+
+<h3 class="govuk-heading-l">Austria</h3>
+
+<div class="call-to-action">
+<p class="govuk-body">The requirements to be a ROPS changed from 6 April 2017. You’ll need to check that the scheme you’re transferring to on or after that date meets the new requirements.<br><br></p>
+
+<p class="govuk-body">HMRC cannot guarantee these are ROPS or that any transfers to them will be free of UK tax. It’s your responsibility to find out if you have to pay tax on any transfer of pension savings. Find out more about <a href="https://www.gov.uk/government/collections/overseas-pension-schemes" class="govuk-link">Overseas pension schemes</a>.</p>
+</div>
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{{
+  govukTable({
+    head: [
+      {
+        text: 'ROPS'
+      },
+      {
+        text: 'Country'
+      }
+    ],
+    rows: [
+      [
+        {
+          text: 'APK Pensionskasse AG'
+        },
+        {
+          text: 'Austria'
+        }
+      ],
+      [
+        {
+          text: 'Bonus Pensionskassen Aktiengesellschaft'
+        },
+        {
+          text: 'Austria'
+        }
+      ],
+      [
+        {
+          text: 'Sozialversicherungspensionskasse AG'
+        },
+        {
+          text: 'Austria'
+        }
+      ],
+      [
+        {
+          text: 'VBV-Pensionskasse Aktiengesellschaft'
+        },
+        {
+          text: 'Austria'
+        }
+      ]
+
+    ]
+  })
+}}
+
+<br /><h3 class="govuk-heading-l">Changes for Austria at 1 July 2018</h3>
```

Comment:
I don't think we should be encouraging people to use `<br>` for whitespace in example code

Reply:
Yes, I agree. Sorry I was taking a bit of a shortcut here as the example is just a visual of the markup for Whitehall.

Having a H3 between two tables has a slightly off visual hierarchy with our current css. Our H3 is closer to the table above (36px) than the table below that it should be at least equally (or more) visually attached to (44px).

I've removed it.